src/lib/chain/offer.ts

```diff
@@ -203,22 +218,27 @@ export function offersInfoToString({
   const offersInfoStr =
     offersInfo.length > 0
       ? `${color.green("Got offers info from chain:")}\n\n${offersInfo
-          .map(
-            ({
-              offerName,
-              offerInfo: { peerCount, minPricePerWorkerEpoch, provider },
-            }) => {
-              return `Offer: ${offerName}
-Provider: ${provider}
-Peer number: ${peerCount}
-Peers: TODO
-
-minPricePerWorkerEpoch: ${minPricePerWorkerEpoch}
-
-effectors: TODO
-`;
-            },
-          )
+          .map(({ offerId, offerName, offerInfo, offerExplorerInfo }) => {
+            return `Offer: ${color.yellow(offerName)}\n${yamlDiffPatch(
+              "",
+              {},
+              JSON.parse(
+                // this is needed to convert all BigInts to string
```

Comment:
Seems a bit hacky. Why not convert them by hand?

Reply:
It's just a lot of values to unpack, it's a lot less code if it's done automatically for you, but I will probably have to do it because there is a price there and it needs to have decimals

Reply:
It's probably not bad, though kinda repetitive. You're clearly doing serialization in a typesafe way. Probably we can create util function for that, which recursively converts bigint to string. Anyway, i'm ok with any approaches keeping in mind our strict time limits :)